client/my-sites/backup/backup-contents-page/style.scss

```
@@ -21,13 +21,44 @@
 		}
 	}
 
+	.status-card__header-content {
+		align-items: center;
+		display: flex;
+		// Given the layout is mobile-first, reverse the div arrangement by default
+		flex-direction: column-reverse;
+	}
+
+	.status-card__learn-about {
+		font-size: $font-body-small;
+		font-weight: 400;
+		text-decoration-line: underline;
+	}
+
+	.status-card__learn-about a {
+		color: var(--Gray-Gray-80, #2c3338);
```

Should we use `--studio-gray-80`?
```suggestion
		color: var(--studio-gray-80 );
```

---

Nice catch. I copied the first color from the Figma design, but indeed `--studio-gray-80` is what's used throughout the code. Updated it.